doc/guides/decoding/decoder_plugin_options.rst

```diff
@@ -0,0 +1,22 @@
+.. _guide_decoder_plugin_opts:
+
+=======================
+Decoder Plugins Options
+=======================
+
+.. currentmodule:: pydicom.pixels.decoders.base
+
+
+.. _decoder_plugin_pydicom:
+
+pydicom
+=======
+
++--------------------------+------------------------------------------------------------------------+
+| Decoder                  | Options                                                                |
++                          +-----------------------+------------------------------------------------+
+|                          | Key                   | Value                                          |
++==========================+=======================+================================================+
+|:attr:`RLELosslessDecoder`| ``rle_segment_order`` | ``"<"`` for big endian segment order (default) |
+|                          |                       | or ``">"`` for little endian segment order     |
```

Comment:
Are the characters correct?  ">" is big endian in Python struct module.

Reply:
Yeah, it should be the other way around. Fixed